tests/ui_tools/test_boxes.py

```diff
@@ -718,7 +718,7 @@ def focus_val(x: str) -> int:
         if write_box.focus_position == write_box.FOCUS_CONTAINER_HEADER:
             write_box.header_write_box.focus_col = (focus_val(
                                                     initial_focus_col_name))
-        write_box.model.get_invalid_recipient_emails.return_value = []
+        write_box.model.check_recipient_validity.return_value = []
```

Comment:
return type specified here doesn't match with that of the functions. :)

Reply:
Right, this commit is just functionality at the moment, because it hasn't been decided on completely. I needed a little feedback on the validation before I proceeded with the tests. Let me know if you have suggestions and I'll modify it.